web/src/app/add-stuff/add-stuff.component.html

```diff
@@ -3,12 +3,17 @@ <h1>Add my stuff</h1>
   <div class="form-group">
     <label for="name">Name</label>
     <input class="form-control" formControlName="name" id="name" required />
+    <div class="alert alert-danger" *ngIf="name.invalid && (name.dirty || name.touched)">
+      <div *ngIf="name.errors.required">
+        Name is required.
+      </div>
+    </div>
   </div>
   <div class="form-group">
     <label for="description">Description</label>
     <input class="form-control" formControlName="description" id="description"/>
   </div>
   <add-category></add-category>
-  <button type="submit">Save</button>
+  <button type="submit" [disabled]="!addStuff.valid">Save</button>
```

Comment:
I don't think we need to disable submit button. :) 
You can do it in the same way it has done in login form: https://github.com/Web-tree/mystuff/blob/89b2f46cb3a782a9edddc46a1b8b4662df48c15e/web/src/app/login/login.component.html#L3
It's better, because you can submit form not only by press button, but by enter or you can bind any other key or event for submit it. So you need to prevent submittion but not pressing the button.